src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace OSPSuite.InstallationValidator.Core.Domain
+{
+   public class StartableProcess
```

Comment:
This seems to be a very generic class that should probably be moved to Core as well?